Simulation control: option to set a max number of transient iterations (#2078)
chaos-polymtl/lethe: doc/source/parameters/cfd/simulation_control.rst

Simulation Control
==================

> This subsection is the most important in a simulation and therefore, the most commonly modified in a parameter file. It controls the general parameters of the simulation, such as the time integration method, end time of a simulation and output settings for paraview files. 

Reviewer: ```suggestion
This subsection is the most important in a simulation and therefore, the most commonly modified in a parameter file. It controls the general parameters of the simulation, such as the time integration method, end time of a simulation and output settings for Paraview files. 
```

    #---------------------------------------------------
    # Transient simulations parameters
    #---------------------------------------------------

>     # The control for the ending of the simulation

Reviewer 2: ```suggestion
    # The control for the end condition of the simulation
```

* ``startup time scaling``: scaling factor used in the iterations necessary to startup the BDF schemes.

.. note::
	SDIRK schemes are self-starting and do not require any additional parameters.

---------------------------------
Transient simulations parameters
---------------------------------

* ``end control``: control for the end condition of a transient simulation. The available options are: 
	* ``iteration``: simulation will end at a specified maximum iteration number.

> 	* ``time`` : simulation will end at a specified maximum time value.

Reviewer 2: ```suggestion
	* ``time``: simulation will end at a specified maximum time value.
```

* ``iteration end``: iteration number at which the simulation ends. Only used when ``end control = iteration``.

* ``time end``: value of the time at which the simulation ends. Only used when ``end control = time``.

.. note::

> 	When ``end control = iteration``, the end time has no effect on the simulation, including on the value of the last time step. Consequently, ``time step independent of end time`` is irrelevant in that case.

Reviewer: ```suggestion
   .. note::
	When ``end control = iteration``, the end time has no effect on the simulation, including on the value of the last time step. Consequently, ``time step independent of end time`` is irrelevant in that case.
```
I think you need to indent the .. note::  here otherwise the documentation does not display it well (it breaks the bullet point lists and splits it into two

> 	When ``end control = iteration``, the end time has no effect on the simulation, including on the value of the last time step. Consequently, ``time step independent of end time`` is irrelevant in that case.

Reviewer: Ohh I like it, great note.
